packages/connector-alipay-native/src/index.ts

```diff
@@ -174,6 +164,18 @@ export default class AlipayNativeConnector implements SocialConnectorInstance<Al
     return { id, avatar, name };
   };
 
+  protected readonly authResponseParser: AuthResponseParser<AuthResponse> = async (
```

Comment:
why not `private`

Reply:
it depends on the LogtoConnector design in another PR. It should be `protected` if this is defined in the parent class.